src/renderer/redux/stores/nodes/reducer.js

```diff
@@ -14,7 +14,7 @@ export default (state = null, action) => {
     case ACTIONS.UPDATE:
       return { ...state, [action.node.id]: action.node };
     case ACTIONS.REMOVE:
-      return deleteSubntl/nodes.messagesTree(action.node.id, state);
+      return deleteSubTree(action.node.id, state);
```

Comment:
👌

Reply:
I don't know what happened there Oo

Reply:
find and replace all ?

Reply:
More like a copy paste in the wrong window